### PR TITLE
setup for Cydia

### PR DESCRIPTION
**Pull request by mwoolweaver**
also adds a simple repo update script

### CODE REVIEW

File: Packages [hunk 1]
```diff
@@ -5,7 +5,7 @@ Maintainer: CoolStar <coolstarorganization@gmail.com>
 Pre-Depends: dpkg (>= 1.14.25-8)
 Depends: apt7-key, apt7-lib (>= 0.7.25.3-7), berkeleydb, cy+cpu.arm64
 Replaces: apt (<< 1:0-1)
-Filename: debs/apt/apt7_0.7.25.3-9_iphoneos-arm.deb
+Filename: ./debs/apt/apt7_0.7.25.3-9_iphoneos-arm.deb
```
Comment (notexe) [inline Packages]: Why change this? `./` only needs to be used when running an executable in the current directory. And this isn't even an executable file. It's literally just a filename for a deb package.

Comment (mwoolweaver) [reply Packages]: dpkg-scanpackages did that. . .

**Comment by nullpixel** [comment]
Okay this actually needs to support gpg signing of the repo

**Comment by mwoolweaver** [comment]
do you have specific gpg key you'd like it signed with?

**Comment by nullpixel** [comment]
it's not made yet, 🔜

**Pull request closed without merging**